packages/sdk/src/utils/Utils.ts

```diff
@@ -31,34 +31,36 @@ export class Utils {
 
     let delimiter = "?"; // initial delimiter
 
-    const cicsPlex = (options && options.cicsPlex) == null ? "" : `${options.cicsPlex}/`;
-    const region = (options && options.regionName) == null ? "" : options.regionName;
+    const cicsPlex = (options && options.cicsPlex) == null ? "" : `${encodeURIComponent(options.cicsPlex)}/`;
+    const region = (options && options.regionName) == null ? "" : encodeURIComponent(options.regionName);
```

Comment:
Are these `== null` for a reason, or can we just do `options?.cicsPlex ? "" : ...`??

```suggestion
    const cicsPlex = options?.cicsPlex ? "" : `${encodeURIComponent(options.cicsPlex)}/`;
    const region = options?.regionName ? "" : encodeURIComponent(options.regionName);
```